src/ResourceManager/ContainerInstance/Commands.ContainerInstance/Commands/NewAzureContainerGroupCommand.cs

```diff
@@ -60,38 +47,45 @@ public class NewAzureContainerGroupCommand : ContainerInstanceCmdletBase
         [Parameter(
             Mandatory = true,
             Position = 1,
-            ParameterSetName = CreateContainerGroupBaseParamSet,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "The container group name.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
         [Parameter(
             Mandatory = true,
-            Position = 1,
-            ParameterSetName = CreateContainerGroupWithRegistryParamSet,
-            ValueFromPipelineByPropertyName = true,
-            HelpMessage = "The container group name.")]
+            Position = 2,
+            HelpMessage = "The container image.")]
+        [ValidateNotNullOrEmpty]
+        public string Image { get; set; }
```

Comment:
Image previously was not a named parameter - was this addition intentional? If so, that is okay, I just want to ensure that the change was intended.

Reply:
Yes.